identity/backend/src/main/java/io/camunda/identity/usermanagement/repository/UserRepository.java

```diff
@@ -24,6 +31,52 @@ public UserRepository(final JdbcTemplate jdbcTemplate) {
 
   public List<CamundaUser> loadUsers() {
     return jdbcTemplate.query(
-        DEF_USERS_QUERY, (rs, rowNum) -> new CamundaUser(rs.getString(1), rs.getBoolean(2)));
+        DEF_USERS_QUERY,
+        (rs, rowNum) ->
+            new CamundaUser(rs.getInt(1), rs.getString(2), rs.getString(4), rs.getBoolean(3)));
+  }
+
+  public void createProfile(final CamundaUser user) {
+    final var id =
+        jdbcTemplate.queryForObject(
+            DEF_USER_ID_QUERY, new Object[] {user.username()}, Integer.class);
+    jdbcTemplate.update(
+        DEF_PROFILE_CREATE,
+        (ps) -> {
+          ps.setInt(1, id);
+          ps.setString(2, user.email());
+        });
+  }
+
+  public CamundaUser loadUser(final String username) {
+    return jdbcTemplate
+        .query(
+            DEF_USER_QUERY,
+            new Object[] {username},
+            (rs, rowNum) ->
+                new CamundaUser(rs.getInt(1), rs.getString(2), rs.getString(4), rs.getBoolean(3)))
```

Comment:
❌  I find this way of retrieving the objects quite challenging to understand, for example, what is `rs.getString(2)`? Can we either switch to retrieving the values by column name or [queryForObject](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/jdbc/core/JdbcTemplate.html#queryForObject(java.lang.String,java.lang.Class))